gamestonk_terminal/stocks/fundamental_analysis/yahoo_finance_view.py

```diff
@@ -147,15 +143,16 @@ def display_calendar_earnings(ticker: str):
     print("")
 
 
-def display_dividends(ticker: str, num: int = 12, export: str = ""):
+def display_dividends(ticker: str, num: int = 12, plot: bool = False, export: str = ""):
```

Comment:
num: int = 10

Reply:
20 represents five years of dividend history assuming quarterly payments. This is my reasoning for chosing this number.